### DISCUSSION TITLE
Warning in the error correction step

### DISCUSSION THREAD

**Discussion by Unaimend** [ablab/spades#1103] [Q&A] [answered]
Hello, I get the following warning after the assembly is already done. What does this warning mean? Can I ignore it?

This is the content of /warnings.log 

=== Error correction and assembling warnings:
 * 0:00:00.111     1M / 26M   WARN   StageManager             (stage.cpp                 : 158)   No saved checkpoint
======= Warnings saved to /work_beegfs/sukem127/datasets/SYNTH181/bachelor_data/all_181_refs/assembly/scripts_long/spades_result_long/warnings.log

Thats my output from the spades.log
[output_spades_hybrid.log](https://github.com/ablab/spades/files/10950626/output_spades_hybrid.log)

**Comment by asl** [comment, marked as answer]
You are using `--continue`, however, there was no checkpoint available. Hence the warning

**Reply by Unaimend** [reply]
ok, that's weird because the warning came I successfully restarted and finished an assembly from a checkpoint.  
Thank you for the quick answer.
